cmd/gogio/main.go

```diff
@@ -34,7 +34,9 @@ var (
 	linkMode      = flag.String("linkmode", "", "set the -linkmode flag of the go tool")
 	extraLdflags  = flag.String("ldflags", "", "extra flags to the Go linker")
 	extraTags     = flag.String("tags", "", "extra tags to the Go tool")
-	iconPath      = flag.String("icon", "", "Specify an icon for iOS and Android")
+	iconPath      = flag.String("icon", "", "specify an icon for iOS and Android")
+	signKey       = flag.String("key", "", "specify the path of the keystore to be used to sign binaries.")
+	signPass      = flag.String("key_pass", "", "specify the password to decrypt the keystore.")
```

Comment:
I think "signpass" is prettier.

What about the key alias? Perhaps rename signpass to signcred ("credentials) and format it like this: "<alias>:<password>"?

Reply:
I never used the `alias`. 🤔

But it should be fine to be `[<alias>]:<password>`, so either `alias:password` or `password` will be accepted. The only problem is if the password itself have `:` character. 🙄

Reply:
I suppose we could leave out alias for now, and add it when needed.